lib/tasks/ff.rake

```diff
@@ -0,0 +1,1076 @@
+namespace :ff do
+  desc "Test AshbyFormFiller"
+  task ashbyhq: :environment do
+    payload = prompt_user(ASHBY_PAYLOADS)
+    Applier::AshbyFormFiller.call(payload)
+  end
+
+  desc "Test BambooFormFiller"
+  task bamboohr: :environment do
+    payload = prompt_user(BAMBOO_PAYLOADS)
+    Applier::BambooFormFiller.call(payload)
+  end
+
+  desc "Test DevitFormFiller"
+  task devitjobs: :environment do
+    payload = prompt_user(DEVITJOBS_PAYLOADS)
+    Applier::DevitFormFiller.call(payload)
+  end
+
+  desc "Test GreenhouseFormFiller"
+  task greenhouse: :environment do
+    payload = prompt_user(GREENHOUSE_PAYLOADS)
+    Applier::GreenhouseFormFiller.call(payload)
+  end
+
+  desc "Test GhFormFiller"
+  task greenhouse_old: :environment do
+    payload = prompt_user(GREENHOUSE_OLD_PAYLOADS)
+    Applier::GhFormFiller.call(payload)
+  end
+
+  desc "Test WorkableFormFiller"
+  task workable: :environment do
+    payload = prompt_user(WORKABLE_PAYLOADS)
+    Applier::WorkableFormFiller.call(payload)
+  end
+end
+
+def prompt_user(group)
```

Comment:
Generally this logic would sit in a service class rather than in the rake file for cleanness - can you move?